gems/sorbet-runtime/test/types/props/decorator.rb

```diff
@@ -362,4 +384,26 @@ class MatrixStruct
     end
     assert_match(/has the word 'secret' in its name/, e.message)
   end
+
+  it 'applies the supplied sensitivity and PII handler' do
+    begin
+      T::Configuration.normalize_sensitivity_and_pii_handler = ->(meta) do
+        meta[:pii] = :set
+        meta[:sensitivity] += 1
+        meta
+      end
+      e = Class.new(T::Struct) do
```

Comment:
Can we define this class at the top of the file, instead of in a `Class.new` hook?